src/SFA.DAS.EmployerAccounts.Jobs/StartupJobs/CreateReadStoreDatabaseJob.cs

```diff
@@ -46,6 +49,8 @@ public async Task Run()
                 }
             };
 
+            _logger.LogInformation("Creating ReadStore database and collection if they don't exist");
+
             await _documentClient.CreateDatabaseIfNotExistsAsync(database);
```

Comment:
I think it would be really useful to log whether the database was created or not, which you can [tell from the response returned](https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.documents.client.documentclient.createdatabaseifnotexistsasync?view=azure-dotnet).

Reply:
Good idea, but the Database object doesn't specify a property which says whether this was created or not. Well not that I can see

Reply:
Added logging to say if the database and document collection were actually created or not.